src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,876 @@
+                        This happens in the first invocation of the <i>upgrade</i> function.
+                        Slot 22 is the <i>isRC</i> flag: <i>cast to-hex 22</i> -> <i>0x16</i>.
+                        Please refer to <i>'Figure 0.1'</i> at the end of this report for the storage layout of OPContractsManager.
+
+  ----- DecodedStateDiff[1] -----
+    Who:               <a href="https://github.com/ethereum-optimism/superchain-registry/blob/c1bcf3601dfdf72f0fd4f5bade180b9c0f94d93b/superchain/configs/mainnet/ink.toml#L63">0x10d7B35078d3baabB96Dd45a9143B94be65b12CD</a>
+    Contract:          DisputeGameFactory - Ink
+    Chain ID:          57073
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x000000000000000000000000c641a33cab81c559f2bd4b21ea34c290e2440c2b
+    Raw New Value:     0x0000000000000000000000004bba758f006ef09402ef31724203f316ab74e4a0
+    Decoded Kind:      address
+    Decoded Old Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L55">0xc641A33cab81C559F2bd4b21EA34C290E2440C2B</a>
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L34">0x4bbA758F006Ef09402eF31724203F316ab74e4a0</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard. DisputeGameFactory contract for 'op-contracts/v2.0.0-rc.1'.
+
+  ----- DecodedStateDiff[2] -----
+    Who:               <a href="https://github.com/ethereum-optimism/superchain-registry/blob/c1bcf3601dfdf72f0fd4f5bade180b9c0f94d93b/superchain/configs/mainnet/ink.toml#L63">0x10d7B35078d3baabB96Dd45a9143B94be65b12CD</a>
+    Contract:          DisputeGameFactory - Ink
+    Chain ID:          57073
+    Raw Slot:          0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e
+    Raw Old Value:     0x0000000000000000000000008d9faaeb46cbcf487baf2182e438ac3d0847f637
+    Raw New Value:     0x0000000000000000000000004699d20479dfd20e15cab4fcb6f2a7cc6fe35443
+    Summary:           Update Permissioned GameType implementation.
+    Detail:            This is gameImpls[1] -> 0x4699d20479dfd20e15cab4fcb6f2a7cc6fe35443
+                       Verify that the old implementation is set in this slot using:
+                        - <i>cast call 0x10d7B35078d3baabB96Dd45a9143B94be65b12CD "gameImpls(uint32)(address)" 1 --rpc-url mainnet</i>
+                        - <i>cast storage 0x10d7B35078d3baabB96Dd45a9143B94be65b12CD 0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e --rpc-url mainnet</i>
+                       The Raw Slot can be derived from:
```

Comment:
Do we need both of these?

Reply:
Not following, both of what?

Reply:
both `cast call` and `cast storage`. They are redundant no?

Reply:
it's fine as is